packages/graphiql/resources/renderExample.js

```diff
@@ -67,7 +67,7 @@ function updateURL() {
 }
 
 function getSchemaUrl() {
-  const isDev = window.location.hostname.match(/localhost$/);
+  const isDev = window.location.hostname.match(/127.0.0.1$/);
```

Comment:
```suggestion
  const isDev = window.location.hostname.match(/127\.0\.0\.1$/);
```
in regex dot means any character